modules/user-data/templates/entrypoint.sh.tpl

```diff
@@ -1,31 +1,47 @@
 #!/usr/bin/env bash
 
-set -euxo pipefail
+set -euo pipefail
 
-echo "Configuring GraphDB instance"
+echo "#################################################"
+echo "#      Begin configuring GraphDB instance       #"
+echo "#################################################"
 
 # Stop in order to override configurations
+echo "Stopping GraphDB"
 systemctl stop graphdb
 
 # Login in Azure CLI with managed identity (user or system assigned)
 az login --identity
 
 # Find/create/attach volumes
 INSTANCE_HOSTNAME=\'$(hostname)\'
-RESOURCE_GROUP=$(curl -H Metadata:true "http://169.254.169.254/metadata/instance/compute/resourceGroupName?api-version=2021-01-01&format=text")
-VMSS_NAME=$(curl -H Metadata:true "http://169.254.169.254/metadata/instance/compute/vmScaleSetName?api-version=2021-01-01&format=text")
-INSTANCE_ID=$(basename $(curl -H Metadata:true "http://169.254.169.254/metadata/instance/compute/resourceId?api-version=2021-01-01&format=text"))
-ZONE_ID=$(curl -H Metadata:true "http://169.254.169.254/metadata/instance/compute/zone?api-version=2021-01-01&format=text")
-REGION_ID=$(curl -H Metadata:true "http://169.254.169.254/metadata/instance/compute/location?api-version=2021-01-01&format=text")
+RESOURCE_GROUP=$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/resourceGroupName?api-version=2021-01-01&format=text")
+VMSS_NAME=$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/vmScaleSetName?api-version=2021-01-01&format=text")
+INSTANCE_ID=$(basename $(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/resourceId?api-version=2021-01-01&format=text"))
+ZONE_ID=$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/zone?api-version=2021-01-01&format=text")
+REGION_ID=$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/location?api-version=2021-01-01&format=text")
```

Comment:
just wondering why we don't do this with the `cli`.

Reply:
This is way faster then using the CLI